src/index.ts

```diff
@@ -112,7 +112,11 @@ function sendTransactions(txs: any[]) {
  * @param parentUrl
  */
 function initSdk(safeAppUrlsRegExp: RegExp[] = []) {
-  config.safeAppUrlsRegExp = [/https:\/\/.*(gnosis-safe\.io|gnosisdev.com)/, ...safeAppUrlsRegExp];
+  config.safeAppUrlsRegExp = [
+    /https:\/\/.*(gnosis-safe\.io|gnosisdev.com)/, // Safe Multisig
+    /http:\/\/localhost:5013/, // Safe Multisig desktop app.
```

Comment:
Why not allow all ports (I had to do this for local development too)

Reply:
Does the desktop app start a https or http server?

Reply:
If we open all ports, perhaps it's better to remove the condition completely. I'm concern about the security problems it could bring. I know it's JS and anything is secure, but at least you could have more control. WDYT?

Reply:
Maybe allow both protocols

Reply:
@matextrem told me it runs on https.

Reply:
I think localhost is a special case where everything could be allowed (chrome has exceptions for localhost development) but I don't mind too much

Reply:
But you whitelisted http here ;)

Reply:
fixed

Reply:
@rmeissner could you approve it? 

I tested it and works fine, we need to create the release tag, so dev-ops could merge it on Monday. We need this published in NPM to make it work desktop app.